theforeman.org/pipelines/vars/foreman/3.1.groovy

```diff
@@ -0,0 +1,41 @@
+def foreman_version = '3.1'
+def foreman_client_distros = [
+    'el8',
+    'el7',
+    'sles12',
+    'sles11'
+]
+def foreman_el_releases = [
+    'el7',
+    'el8'
+]
+def foreman_debian_releases = ['buster', 'bionic', 'focal']
+
+def pipelines_deb = [
+    'install': [
+        'debian10',
+        'ubuntu1804',
```

Comment:
we are dropping ubuntu1804, so I think we dont require 'ubuntu1804'? @ekohl am I correct?